tests/step8_macros.mal

```diff
@@ -173,3 +173,12 @@ x
 (->> [4] (concat [3]) (concat [2]) rest (concat [1]))
 ;=>(1 3 4)
 
+;; Test that defining a macro does not mutate an existing function.
+(def! f (fn* [x] (number? x)))
+;=>#<function (x) -> (number? x)>
+(defmacro! m f)
+;=>#<macro (x) -> (number? x)>
```

Comment:
The way that macros and functions render is up to the implementation author. I would just remove the two lines that test the rendering of functions and macros (runtest will just consider them passing tests no matter what happens). The following two tests are the interesting part anyways. And that's a nice concise way to distinguish between macros and functions!